components/applications-service/cmd/applications-service/commands/root.go

```diff
@@ -34,9 +34,20 @@ func init() {
 	RootCmd.PersistentFlags().String("key", "", "path to key")
 	RootCmd.PersistentFlags().String("cert", "", "path to cert")
 
-	viper.BindPFlag("cert", RootCmd.PersistentFlags().Lookup("cert"))
-	viper.BindPFlag("root-cert", RootCmd.PersistentFlags().Lookup("root-cert"))
-	viper.BindPFlag("key", RootCmd.PersistentFlags().Lookup("key"))
+	// https://github.com/spf13/viper/blob/7a605a50e69cd1ea431c4a1e6eebe9b287ef6de4/viper.go#L939
+	// err is non-nil when the flag doesn't exist. panic here should get caught quickly by Ci.
```

Comment:
We could've done a `//nolint: errcheck` here, but I opted to make it panic as this should make it impossible for us to pass Ci if the flag names get out of sync.